blogposts/2021/release-3.25.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Sourcegraph 3.25 release"
+publishDate: 2021-02-20T10:00-07:00
+description: "Sourcegraph 3.25 is released."
+tags: [blog, release]
+slug: "release/3.25"
+published: true
+changelogItems:
+- description: "Go `1.15` introduced changes to SSL/TLS connection validation which requires certificates to include a `SAN`. This field was not included in older certificates and clients relied on the `CN` field. You might see an error like `x509: certificate relies on legacy Common Name field`. We recommend that customers using Sourcegraph with an external database that is connected using SSL/TLS check whether the certificate is up to date. AWS RDS customers please reference [AWS' documentation on updating the SSL/TLS certificate](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.SSL-certificate-rotation.html) for steps to rotate your certificate."
+  category: Admin
+  url: https://github.com/sourcegraph/sourcegraph/pull/18024
+# Added
+- description: "New site config option `\"log\": { \"sentry\": { \"backendDSN\": \"\u003cREDACTED\u003e\" } }` to use a separate Sentry project for backend errors."
+  url: https://github.com/sourcegraph/sourcegraph/pull/17363
+  category:
+- description: Structural search now supports searching indexed branches other than default.
+  url: https://github.com/sourcegraph/sourcegraph/pull/17726
+  category: Search
+- description: Structural search now supports searching unindexed revisions.
+  url: https://github.com/sourcegraph/sourcegraph/pull/17967
+  category: Search
+- description: "New site config option `\"allowSignup\"` for SAML authentication to determine if automatically create new users is allowed."
+  url: https://github.com/sourcegraph/sourcegraph/pull/17989
+  category:
+- description: "Experimental: The webapp can now stream search results to the client, improving search performance. To enable it, add `{ \"experimentalFeatures\": { \"searchStreaming\": true } }` in user settings."
+  url: https://github.com/sourcegraph/sourcegraph/pull/16097
+  category: Search
+- description: "New product research sign-up page. This can be accessed by all users in their user settings."
+  url: https://github.com/sourcegraph/sourcegraph/pull/17945
+  category: Search
+- description: "New site config option `productResearchPage.enabled` to disable access to the product research sign-up page."
+  url: https://github.com/sourcegraph/sourcegraph/pull/17945
+  category: Search
+- description: Pings now contain Sourcegraph extension activation statistics.
```

Comment:
I recommend cutting this.

Reply:
I'm seeing three lines for `Pings now contain...` in the preview. Can they be consolidated into one? I don't know enough about the feature/impact on user to say if we should cut them or not - your call!